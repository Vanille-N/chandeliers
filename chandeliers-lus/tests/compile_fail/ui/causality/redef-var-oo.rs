chandeliers_lus::decl! {
    node foo() returns (a : bool; a : int);
    let a = 1; tel
}

fn main() {}
