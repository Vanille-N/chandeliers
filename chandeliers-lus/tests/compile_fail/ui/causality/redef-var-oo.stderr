error: Attempt to redefine a, when a prior item already defines it
 --> tests/fail/ui/causality/redef-var-oo.rs:2:35
  |
2 |     node foo() returns (a : bool; a : int);
  |                                   ^
  |
note: Already defined here
 --> tests/fail/ui/causality/redef-var-oo.rs:2:25
  |
2 |     node foo() returns (a : bool; a : int);
  |                         ^
